uaclient/cli.py

```diff
@@ -1181,8 +1181,17 @@ def action_collect_logs(args, *, cfg: config.UAConfig):
             ),
         )
 
-        for log in ua_logs:
+        extra_logs = ("/etc/cloud/build.info",)
```

Comment:
Let's add this extra log into the `ua_logs` variable.
Also, let's create a constant for it on `defaults.py`

Reply:
Since you are touching this file, let's also replace `/etc/ubuntu-advantage/uaclient.conf` on that variable to `DEFAULT_CONFIG_FILE` that is also present on `defaults.py`

Reply:
On it, I was not sure to separate the extra logs or not. But having the paths in defaults.py will be better to maintain.